Makefile

```diff
@@ -12,7 +14,7 @@ pdf: *.md
 				 -V links-as-notes=True \
 				 -f markdown \
 				 -t latex \
-				 *.md \
+				 !(*README).md \
```

Comment:
Previously, building locally includes the README.md in the pdf printout when building.  This will prevent the README.md from being included in future builds.  Downside is it requires BASH to be used with extended globbing enabled.

Reply:
This is no longer needed, see the subsequent commit added to this branch.